CONTRIBUTING.md

```diff
@@ -314,7 +314,7 @@ Your PR should include:
 
 ## Local Development Setup
 
-The recommended way to develop and test Ambient Code Platform locally is using OpenShift Local (CRC - CodeReady Containers). This provides a complete OpenShift environment running on your local machine with real authentication, RBAC, and production-like behavior.
+The recommended way to develop and test Ambient Code Platform locally is using **Minikube**. This provides a lightweight Kubernetes environment on your local machine with no authentication requirements, making development fast and easy.
 
 ### Installing and Setting Up CRC
```

Comment:
The changes to CONTRIBUTING.md seem incomplete here.  It now recommends Minikube instead of CRC but then it dives right into how to install and setup CRC.